flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -151,7 +159,7 @@ public static NetworkEnvironment create(
 			checkNotNull(taskEventPublisher),
 			resultPartitionFactory,
 			singleInputGateFactory,
-			resultPartitionBufferPoolFactory);
+			resultPartitionBufferPoolFactory, inputGateBufferPoolFactory);
```

Comment:
inputGateBufferPoolFactory should be in separate line for the commit of `[hotfix][network] Introduce InputGateBufferPoolFactory`